apps/acrs/behaviours/limit-brother-sister.js

```diff
@@ -0,0 +1,17 @@
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    locals.siblingCount = '1';
+    if(req.sessionModel.get('referred-siblings') !== undefined) {
+      const referredSiblings = req.sessionModel.get('referred-siblings').aggregatedValues;
+      const siblingsLimit = referredSiblings.length >= req.form.options.aggregateLimit;
+      const siblingCount = referredSiblings.length + 1;
```

Comment:
Do you mean to add one here as `.length` is the count of entries in the array?

Reply:
`siblingCount` is the local variable used in the title of a new sibling being added, so the number of the new sibling to be added is the current number of referred plus one.